metagpt/provider/bedrock/bedrock_provider.py

```diff
@@ -57,15 +57,52 @@ def get_choice_text_from_stream(self, event) -> str:
 
 
 class CohereProvider(BaseBedrockProvider):
-    # See https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-cohere-command.html
+    # For more information, see
+    # (Command) https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-cohere-command.html
+    # (Command R/R+) https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-cohere-command-r-plus.html
+
+    def __init__(self, model_name: str) -> None:
+        self.model_name = model_name
 
     def _get_completion_from_dict(self, rsp_dict: dict) -> str:
         return rsp_dict["generations"][0]["text"]
 
+    def messages_to_prompt(self, messages: list[dict]) -> str:
+        if "command-r" in self.model_name:
```

Comment:
Due to a account limit, can you put some passed logs under using commnd-r/jamba models.

Reply:
Can you elaborate on this one?

Reply:
The log of running commnd-r/jamba models like executing `examples/hello_world.py` through asking `who are you`.

Reply:
I don't understand what you're getting at and looking at `exmaples/hello_world.py` didn't help. The other `message_to_prompt` implementations are not logging anything anyway. Feel free to edit this PR directly if it helps.

Reply:
I don't have a aws account to test command-r/jamba models. So in the previous comment, I wanted you to post some test results(directly using `examples/hello_world.py`) to show that it works.